### PR TITLE
Popup menu

### PR DESCRIPTION
**Pull request by AbuTalha3**
In this branch:-
- I have added popup.js file and popup pages/cards of projects
- Now the portfolio page is more interactive.
- Webhint issue cannot be resolved.
- Learning to use JavaScript with CSS and Html is cool.

### CODE REVIEW

File: menu.js
Comment (Efiamotu-1) [inline menu.js:1]: - [ ] On mobile devices, the spacing and alignment of the popup contents vertically is not the same as can be seen in the figma image below. I have used red lines to show you so you can know how to adjust them. I kindly suggest you make the required and necessary changes so your design looks great :+1: :sparkles: 
- figma design
ex-ap
- your design
exap

Comment (Efiamotu-1) [inline menu.js:1]: - [ ] You are not using the appropriate color for the tilte of the project as shown in the figma image below please kindly adjust your design for the text. :+1: 
- figma design
ftn
- your design
tfn

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @AbuTalha3 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi  Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**